testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/security/auditing/SecurityAuditingTestCase.java

```diff
@@ -69,50 +72,58 @@
 @RunWith(Arquillian.class)
 @RunAsClient
 @ServerSetup({SecurityAuditingTestCase.SecurityAuditingTestCaseSetup.class})
-@Ignore("[WFLY-15263] Update for lack of legacy security.")
 public class SecurityAuditingTestCase extends AnnSBTest {
 
     private static final Logger log = Logger.getLogger(testClass());
 
     private static File auditLog = new File(System.getProperty("jboss.home", null), "standalone" + File.separator + "log" + File.separator + "audit.log");
 
-    static class SecurityAuditingTestCaseSetup extends AbstractMgmtServerSetupTask {
+    private static final boolean USE_ELYTRON = System.getProperty("elytron") != null;
```

Comment:
No need to update this PR just for this, but I assume this line is cruft.

Reply:
Yeah, good catch, that line isn't needed here. I've removed it.

Reply:
Thanks. I'm glad I asked as something I did later led to https://github.com/wildfly/wildfly/pull/15211. :)